CHANGELOG.md

```diff
@@ -2,7 +2,34 @@
 
 ## Version 0.41.0
 
-_In development._
+([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.40.0...v0.41.0))
+
+### Highlights
+
+📓 The Layerwise Richardson Extrapolation **(LRE) user guide is complete**!
+The user guide contains information about both the ins and outs of using the implementation, as well as covering the theory behind the technique so you can make judgements about when to apply the technique.
+In addition to finishing the user guide, we also have a new tutorial comparing both the performance and overhead needed for LRE and ZNE.
+
+📹 As part of launching LRE we made a **short tutorial video** to showcase the technique, along with how to use it.
+Check it out [here](https://www.youtube.com/watch?v=47GWi4h7TWM)!
+
+🧑‍🔬 **First time contributor** @jpacold recreated results from [a paper](https://arxiv.org/abs/2211.08318) on phase transitions in the Ising model.
+Both the paper authors, and @jpacold both used Mitiq's ZNE module to apply error mitigation.
+This is both an informative tutorial on turning physics problems into something amenable on a quantum computer, and a class in applying error-mitigation.
+
+#### ✨ Enhancements
+
+- LRE functionality now works with all supported frontends (instead of just cirq)
+
+#### 🧑🏽‍💻 Developer Improvements
+
+- remove failing test; simplify layerwise ZNE tests (#2545) [@natestemen]
+
+### 📞 Call for ideas
+
+We're currently looking into what features we could add to make Mitiq more **noise-aware**.
+If you have ideas and features requests in this area, do get in touch!
+Contact Nate at `nate@unitary.fund`.
```

Comment:
I would direct people to a github discussion instead.

Reply:
+1 for directing to a github discussion. I also like the newsletter idea, we should add a call for ideas to the next edition.

Reply:
Good call. Replaced with a link to the discussion (https://github.com/unitaryfund/mitiq/pull/2557/commits/512529c3283a4a9a5f91c98e6ef18105224d61ef), and will add a note in the future newsletter.